src/components/Dashboard/TodoList.vue

```diff
@@ -521,6 +520,52 @@
                   <!-- no action button in this case -->
                 </template>
 
+                <!-- authorization approved continuation filing -->
+                <template v-else-if="EnumUtilities.isStatusApproved(item)">
+                  <v-btn
+                    class="btn-draft-resume"
+                    color="primary"
+                    :disabled="!item.enabled"
+                    @click.native.stop="doResumeFiling(item)"
+                  >
+                    <span>Resume</span>
+                  </v-btn>
+                  <!-- dropdown menu -->
+                  <v-menu
+                    offset-y
+                    left
+                  >
+                    <template #activator="{ on }">
+                      <v-btn
+                        id="menu-activator"
+                        class="actions__more-actions__btn px-0"
+                        color="primary"
+                        :disabled="!item.enabled"
+                        v-on="on"
+                      >
+                        <v-icon>mdi-menu-down</v-icon>
+                      </v-btn>
+                    </template>
+                    <v-list class="actions__more-actions py-2">
+                      <v-list-item
+                        id="btn-delete-application"
+                        @click.stop="confirmDeleteApplication(item)"
+                      >
+                        <v-icon
+                          class="pr-1"
+                          color="primary"
+                          size="18px"
+                        >
+                          mdi-delete-forever
+                        </v-icon>
+                        <v-list-item-title>
+                          Delete {{ EnumUtilities.filingTypeToName(item.name) }}
```

Comment:
No. Once the authorization is submitted (ie, a review record is created), the user can no longer delete the filing.

The user can only delete the filing when they're at the authorization stage (including save and resume later) and before they submit it for review.

Reply:
Removed the drop down menu